CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added support for multiple content shares in meetings through the `ContentShareProvider`. The provider now accepts a `maxContentShares` prop (default: 1, range 1-2) to specify the maximum number of concurrent content shares allowed.
```

Comment:
nonblocker: `maxConcurrentContentShare` may be a better name?

Reply:
I feel 'concurrent' is redundant here, since 'maxContentShares' itself is already clear and unambiguous.

Reply:
What if we dont have such a prop? Can we take Video tiles like route. For example, we have a max limit on video streams but we dont have a prop for the max limit for them limiting builders. Correct me if not here. Have to check how that is limited component usage wise.

Reply:
Right, just trying to see whether we adding a new prop vs using the `tiles.length` is fine too. If the later is fine, why introduce a new one? Now still going through review, will understand and sync on the breaking change part.

Reply:
Right I get it now, its more like a feature flag in a way where its honored and backward compatible in the `ContentShareProvider`.